### ISSUE TITLE
Nightly builds not working

### ISSUE THREAD

**Issue by renatoc8** [madskristensen/vswebessentials.com#75]
I recently noticed that I hadn’t received any updates for the Web Essentials 2013 nightly builds in a while.
When I browsed to the URL (http://vswebessentials.com/nightly/feed.ashx) in my browser, I noticed that the server was reporting a web.config error.

**Comment by GProulx** [comment]
Hi @madskristensen 

He's right about the nightly build not updated since January 16th. Now that you added AppVeyor to WE2013 and 2015 [it can be possible to use that to automatically push](http://www.appveyor.com/docs/how-to/git-push) updated vsix after each build on the vswebessentials.com repository.

**Comment by madskristensen** [comment]
Yeah, you can grab it here https://ci.appveyor.com/project/madskristensen/webessentials2013/build/artifacts

**Comment by GProulx** [comment]
Nice! I will make required modifications to feed.ashx to access those files (for both 2013 and 2015) instead if it's ok for you?

**Comment by madskristensen** [comment]
I don't think you can. The url for the actual VSIX changes from build to build

**Comment by madskristensen** [comment]
for VS2015 it's easier because it will be right here http://vsixgallery.com/extension/5fb7364d-2e8c-44a4-95eb-2a382e30fec9/ when AppVeyor upgrades to VS2015 RTM soon

**Comment by GProulx** [comment]
You're right about the variable path. So the proposition of using AppVeyor to push it to the VSWebEssentials repository could be use.

For your proposition for VS2015... you really want to push to the gallery after every merge? You trust _us_ more that I would! :smile:

**Comment by madskristensen** [comment]
Yeah, I think having a publicly available CI build is important. That's why I built vsixgallery.com to begin with

**Comment by GProulx** [comment]
I totally agree about the publicly available CI build!

For my comment it was my mistake, I mixed up *your* vsixgallery with the [Visual Studio Gallery](https://visualstudiogallery.msdn.microsoft.com/ee6e6d8c-c837-41fb-886a-6b50ae2d06a2) where, I think you would agree, it's not the perfect place for a nightly build. :smile: 

Do you have plans to put back the <code> | Vsix-PublishToGallery</code> part for WE2013 too? I could:
* update the Nightly builds section to the Web site accordingly for 2013 and 2015 after the RTM
* Do a little bit of cleanup on the feed folder for old vsix
* Force a redirect to the new feed page for a while
